### ISSUE TITLE
Enable virus checking

### ISSUE THREAD

**Issue by jayanthdeejay** [DigitalWPI/digitalwpi#46]
Add Clamav gem to Gemfile if it isn't already added and add file uploads to be scanned by clamav. Test with an infected [file](https://www.eicar.org/86-0-Intended-use.html).

**Comment by jayanthdeejay** [comment]
I have looked at the [hyrax](https://github.com/samvera/hyrax/wiki/Hyrax-Management-Guide#virus-checking) virus checking. Enabled the same and tested with a malware file. It did not recognize the file as infected and went on to add the infected file. 

Came across [this](https://github.com/samvera/hyrax/issues/2284) and after going through the issue, I realized the built in virus scanning has inherent issues with it. Also, looked at the UC's fix to the inherent empty file issue, tested the same locally. For some reason, even that didn't work. Tested with EICAR sample malware file but in both the cases, the application failed to recognize the file as infected. Since our application is not a self-submission repository, assuming most of our files will be ingested, I am thinking that we can close this issue. @awscott Let me know what you think.
